### ISSUE TITLE
Test: Simple settings indicators accessibility

### ISSUE THREAD

**Issue by rzhao271** [microsoft/vscode#167084]
Refs: #159088

- [x] macOS @mjbvz
- [x] linux @chrmarti
- [x] windows @joyceerhl

Complexity: 3

[Create Issue](https://github.com/microsoft/vscode/issues/new?body=Testing+%23167084%0A%0A&assignees=rzhao271)

---

In the Settings editor, there are indicators, which are labels that show up in parentheses next to some settings such as "Not synced".

1. Enable Settings sync.
2. Open the Settings editor.
3. Search `emmet`.
4. Navigate from the search box to the `emmet.extensionsPath` setting using the keyboard.
5. Tab to the "Not synced" indicator. The entire indicator should be outlined.
6. Once focused on an indicator, press space or enter to open the hover and focus it.
7. Alternatively, one should be able to hover over an indicator with a mouse, and then click on the hover to focus it.
8. Pressing any key while the hover is focused should close the hover. Focus should return back to the element that had focus before the hover was focused.
9. The indicator and hover should be navigable with a screenreader.
10. Steps 3 to 9 should also work when searching `@lang:go editor spaces`, tabbing down to the `editor.insertSpaces` setting, and tabbing to the "Default value overridden" indicator.
